pkgs/top-level/python-packages.nix

```diff
@@ -11173,6 +11175,8 @@ self: super: with self; {
 
   pkuseg = callPackage ../development/python-modules/pkuseg { };
 
+  pyluach = callPackage ../development/python-modules/pyluach { };
+
   plac = callPackage ../development/python-modules/plac { };
 
```

Comment:
This still isn't sorted alphabetically, `pyluach` should be further down, shouldn't it?